gcp_variant_transforms/beam_io/vcfio_test.py

```diff
@@ -509,6 +512,210 @@ def test_dynamic_work_rebalancing(self):
+
+  def test_write_bzip2_file_auto(self):
+    self.path = tempfile.NamedTemporaryFile(suffix='.vcf.bz2').name
+    sink = VcfSink(self.path, compression_type=CompressionTypes.AUTO)
+    self._write_lines(sink, self.variants)
+
+    with bz2.BZ2File(self.path, 'r') as f:
+      for actual, expected in zip(f.read().splitlines(), self.variant_lines):
+        self._assert_variant_lines_equal(actual, expected)
+
+  def test_write_gzip_file(self):
+    sink = VcfSink(
+        self.path, compression_type=CompressionTypes.GZIP)
+    self._write_lines(sink, self.variants)
+
+    with gzip.GzipFile(self.path, 'r') as f:
+      for actual, expected in zip(f.read().splitlines(), self.variant_lines):
+        self._assert_variant_lines_equal(actual, expected)
+
+  def test_write_gzip_file_auto(self):
+    self.path = tempfile.NamedTemporaryFile(suffix='.vcf.gz').name
+    sink = VcfSink(self.path, compression_type=CompressionTypes.AUTO)
+    self._write_lines(sink, self.variants)
+
+    with gzip.GzipFile(self.path, 'r') as f:
+      for actual, expected in zip(f.read().splitlines(), self.variant_lines):
+        self._assert_variant_lines_equal(actual, expected)
+
+  def test_write_gzip_file_empty(self):
+    sink = VcfSink(
+        self.path, compression_type=CompressionTypes.GZIP)
+    self._write_lines(sink, [])
+
+    with gzip.GzipFile(self.path, 'r') as f:
+      self.assertEqual(f.read().splitlines(), [])
+
+  def test_write_text_file_with_header(self):
+    headers = ['header1\n', 'header2\n']
+    sink = VcfSink(self.path, headers=headers)
+    self._write_lines(sink, self.variants)
+
+    with open(self.path, 'r') as f:
+      lines = f.read().splitlines()
+      self.assertEqual(lines[0], 'header1')
+      self.assertEqual(lines[1], 'header2')
+      for actual, expected in zip(lines[2:], self.variant_lines):
+        self._assert_variant_lines_equal(actual, expected)
+
+  def test_write_text_file_empty_with_header(self):
+    headers = ['header1\n', 'header2\n']
+    sink = VcfSink(self.path, headers=headers)
+    self._write_lines(sink, [])
+
+    with open(self.path, 'r') as f:
+      self.assertEqual(f.read().splitlines(), ['header1', 'header2'])
+
+  def test_write_dataflow(self):
+    pipeline = TestPipeline()
+    pcoll = pipeline | beam.core.Create(self.variants)
+    pcoll | 'Write' >> vcfio.WriteToVcf(self.path)  # pylint: disable=expression-not-assigned
+    pipeline.run()
+
+    read_result = []
+    for file_name in glob.glob(self.path + '*'):
+      with open(file_name, 'r') as f:
+        read_result.extend(f.read().splitlines())
+
+    for actual, expected in zip(read_result, self.variant_lines):
+      self._assert_variant_lines_equal(actual, expected)
+
+  def test_write_dataflow_auto_compression(self):
+    pipeline = TestPipeline()
+    pcoll = pipeline | beam.core.Create(self.variants)
+    pcoll | 'Write' >> vcfio.WriteToVcf(self.path + '.gz',  # pylint: disable=expression-not-assigned
```

Comment:
You can actually just assign this to '_' to get rid of the lint error...e.g. _ = (pcoll | ...)

Reply:
Done.